discovery.md

```diff
@@ -215,19 +214,29 @@ The following sections define the attributes that appear in a Service entity.
   specification makes no statement, or guarantees, as to the forwards or
   backwards compatibility of a Service as it changes over time.
 
-  Note, unlike the `name` attribute which only needs to be unique within the
-  scope of a single Discovery Endpoint, the global uniqueness aspect of this
-  attribute allows for the discovery of the same Service across multiple
-  Discovery Endpoints.
-
   See the Primer for more information.
 
 - Constraints:
   - REQUIRED in responses from the Discovery Endpoint.
-  - MUST be a valid UUID per RFC4122.
 - Examples:
   - `bf5ff5cc-d059-4c79-a89a-2513e45a1340`
 
+#### authority (service authority)
+
+- Type: `URI`
+- Description: Identifies the authority for this service. Similar to schemas authority.
+  Note, the pair `authority` and `id` are globally unique
+- Constraints:
+  - OPTIONAL. If the attribute is absent or empty, its implied default value is the base
```

Comment:
The OPTIONAL and REQUIRED aspects here are a bit awkward. Perhaps just say that when created/imported, a missing values MUST be assigned the base URI. Then we never need to worry about it being "empty".

I do wonder though, is it possible the same DE can have more than one URL associated with it? Which base would be used? Does it matter?

Reply:
I think this was copied from [here](https://github.com/cloudevents/spec/blob/master/schemaregistry/schemaregistry.md#authority-schema-authority). We would need something corresponding to `getSchemaVersionByURI` for the discovery API.

Reply:
It can probably happen if vhost are used in front, but I do not think it matters as they will be treated as different services